sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -1787,6 +1792,53 @@ public EditEventSteps(
           fillStartDataDE(eventStartDate);
         });
 
+    Then(
+        "^I Verify The Eye Icon opening the Map is disabled in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsDisabled(EYE_ICON);
+        });
+
+    Then(
+        "^I Verify The Eye Icon opening the Map is enabled in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsEnabled(EYE_ICON);
+        });
```

Comment:
compress these 2 methods under a single one and make the enabled/disabled value as parameter